First commit (v1)

TRSV-egor opened the pull request:
Сдача домашнего задания "Консольное приложение №1"

Commit:
First commit (v1)

DmitrijEgorow commented on src/main/java/Input.java:
```diff
@@ -0,0 +1,82 @@
+    }
+
+    //Запрос товаров у пользователя и добавление их в калькулятор
+public static void getAndAddPositions(Calc calc){
```
Перед коммитом стоит приводить файлы к правильному стилю, так как код будут смотреть другие разработчики. В IDEA это легко делать путём вызова горячей комбинации клавиш (она зависит от ОС), её можно посмотреть в меню Code → Reformat Code

DmitrijEgorow commented on src/main/java/Product.java:
```diff
@@ -0,0 +1,11 @@
+//Используется для хранения позиции товара с ценой
+public class Product {
+    String product;
+    Double price;
```
Можно и примитивный тип использовать здесь

Commit:
Second commit (v1.1):
1) исправлен тип переменной в классе Product с Double на double 2) Выполнено переформатирование кода во всех файлах 3) исправлен механизм подстановки "рублей"

Commit:
Third commit (v1.11):
исправлен тип переменной в конструкторе класса Product с Double на double
